conditional/blueprints/member_management.py

```diff
@@ -603,6 +630,32 @@ def clear_active_members():
     return jsonify({"success": True}), 200
 
 
+@member_management_bp.route('/manage/export_active_list', methods=['GET'])
+def export_active_list():
+    sio = io.StringIO()
+    csvw = csv.writer(sio)
+
+    active_list = [["Full Name", "RIT Username", "Amount to Charge"]]
+    for member in ldap_get_active_members():
+        full_name = member.cn
+        rit_username = re.search(".*uid=(\\w*)", member.ritDn).group(1)
+        will_coop = CurrentCoops.query.filter(
+            CurrentCoops.date_created > start_of_year(),
+            CurrentCoops.semester != "Neither",
```

Comment:
I don't think it is, now that you mention it. I was modeling my query after [this one in the /manage route](https://github.com/ComputerScienceHouse/conditional/blob/develop/conditional/blueprints/member_management.py#L72), where they use that. I'll fix this query in a new commit